client/clientui.py

```diff
@@ -41,7 +41,8 @@ def __init__(self, master, client, queue, send_command):
         # pprint(vars(master))
         self.side_bar = master.children['side_bar']
         self.output_panel = master.children['output_frame'].children['output']
-        # self.output_panel = self.output_frame.children['output']
+        self.output_panel.configure(state="normal")
+        self.output_panel.bind('<Key>',lambda e: 'break')
```

Comment:
OK, what exactly does this line do? I like to understand the code that gets added.
Does it just interrupt key entry on this widget and break out of the function to prevent it?

 Also why does it need to be here instead in the `draw_output` method? I'm just being picking about what handles what on this point.